openstack_controller/tests/conftest.py

```diff
@@ -989,7 +989,7 @@ def get(url, *args, **kwargs):
         method = 'GET'
         url = get_url_path(url)
         if http_error and url in http_error:
-            raise requests.exceptions.HTTPError(response=http_error[url])
+            return http_error[url]
```

Comment:
Why do we stop raising an exception here?

Reply:
We're setting http_error with MockResponses. Later we handle this response and raise an exception if needed with `raise_for_status`